### DISCUSSION TITLE
PS5 and 4 Controllers not being detected to pair to Switch

### DISCUSSION THREAD

**Discussion by Lolket123** [ndeadly/MissionControl#674] [General]
i have downloaded and installed the 0.9.2 version of mission control onto my micro SD card, the one in which holds my main modded files for the switch. unfortunately, I recently contracted a case of stick drift on my joy cons and really wanted to use a different controller, that's when I found this mod "mission control" which should supposedly allow me to connect both my ps4 or ps5 controller onto my switch. unfortunately no matter what I do I cant seem to get it to work. i have spent a good hour and 30 mins constantly putting my controllers into pare mode waiting for them to pop up on the switch screen saying "paired" but alas, that hasn't happened. This prompted me to do a little research on this mod and see how effective it is at working. to my surprise the majority of people who use this say it works great and it works fine, so this lead me to the conclusion that I may have installed the mod wrong, so re-installed it. and again it didn't work. so i thought to myself, mabey it was a AMS compatibility issue, so i tried and older version and it still didn't work. This leads me to this github post searching for some amazing soul in the community to help me out since i honestly don't know how to get this thing to work :) anyways sorry for the long paragraph but i find it helps read big chunks of text when they are set out in a narrative format. anyways look forward to reading any answers :)

**Comment by ndeadly** [comment]
Assuming you've followed the install instructions correctly and rebooted your console etc, try fixing your archive bits in hekate.
